src/generate-schema.js

```diff
@@ -64,8 +64,11 @@ module.exports = {
                 var schema = schemas[name];
                 //thank you MS for messing around with filenames!
                 var filename = normPath(schema.extra.filename);
-                var source = filename.toLowerCase().startsWith(normPath(deps).toLowerCase())
-                    ? (relDeps + '/ts/' + path.dirname(filename.substring(filename.lastIndexOf('-api-dependencies/ts/') + 21, filename.lastIndexOf('/ts')+3)) + '/json-schema-v4/') : '';
+                var relativeDir = params.isCustomDependencyPath() ?
+                    (relDeps + '/json-schema-v4' + path.dirname(filename.substring(deps.length + 3)) + '/') :
+                    (relDeps + '/ts/' + path.dirname(filename.substring(filename.lastIndexOf('-api-dependencies/ts/') + 21, filename.lastIndexOf('/ts')+3)) + '/json-schema-v4/');
+                relativeDir =
```

Comment:
`relativeDir = `

What is this line for (line 70)? Makes no sense to me.

Reply:
Yep, that was stuff that I accidentally committed... argh.
It is fixed now on develop.